addon/components/frost-list.js

```diff
@@ -150,11 +152,12 @@ export default Component.extend({
       })
     }
 
-    $(document).on(`keyup.${this.elementId} keydown.${this.elementId}`, this.setShift.bind(this))
+    this._keyHandler = this.setShift.bind(this)
+    $(document).on(`keyup.${this.elementId} keydown.${this.elementId}`, this._keyHandler)
```

Comment:
I assume there are no scenarios in which the `elementId` can change during a components life cycle? Otherwise this would be problematic as the `.off()` could be for different events than `.on()`.